src/lib/__tests__/messenger.test.js

```diff
@@ -0,0 +1,156 @@
+import expect, { spyOn, restoreSpies } from 'expect';
+import TelegramBot from 'node-telegram-bot-api';
+import messengerFactory from '../messenger';
+import { commands } from '../../handlers/inputHandler';
+import config from '../../config';
+
+describe('Messenger', () => {
+  let messenger;
+  let bot;
+
+  beforeEach(() => {
+    bot = new TelegramBot(config.telegram.token, { polling: true });
+    messenger = messengerFactory(bot);
+  });
+
+  describe('listen', () => {
+    it('binds text handler to on text event', () => {
+      const spy = spyOn(bot, 'on');
+      messenger.listen();
+      expect(spy).toHaveBeenCalledWith('text', messenger.handleText);
```

Comment:
I absolutely agree with you on this one, however this is just to quickly get somewhat of a coverage in order not to break too much existing behavior. 

I can either add an issue to say that it should be fixed better and we merge this for now, or I'll fix it directly, what do you think?